OpenRA.Game/FileSystem/InstallShieldCABExtractor.cs

```diff
@@ -404,9 +405,44 @@ public string DirectoryName(uint index)
 			return test;
 		}
 
+		public string GetFirstSubDirectory(string package)
```

Comment:
>German disks has 13 cabinets. so i decided to check for data1.hdr only.

TLDR he's hacking it because German and non-german TFD contents is different.
Which, again, is the exact reason why people have been saying we shouldn't try to support them.